src/components/CurrencyLogo/index.tsx

```diff
@@ -69,8 +70,18 @@ export default function CurrencyLogo({
     }
     return undefined
   }, [checkSummed, polygon])
+
   const uriLocationsPOlygon = useHttpLocations(polygonURI)
 
+  const celoURI = useMemo(() => {
+    if (checkSummed && celo?.[checkSummed]) {
+      return celo?.[checkSummed].token.logoURI
+    }
+    return undefined
+  }, [checkSummed, celo])
+
```

Comment:
rm this added line to group the celo uri variables together (for consistency with above chains)